app/assets/stylesheets/era_custom.css.scss

```diff
@@ -488,6 +488,19 @@ body.home h4{
 	color: $link1Hover;
 }
 
+.avalon>a{
```

Comment:
Yeah theres so much CSS work that could be done here from looking at this file. Lots of repeated/duplication of classes and styles.

`.copyright` and `.avalon` ones can be merged...maybe do it to a `.footer a` class instead? I highly doubt you need the child selector as well.

Anyways lots of improvements, depends how much you want to chew off. But yes...take advantage of SASS. Maybe read some quick tutorials? Highly useful and much more readable I'd say.